ui/imports/utils/LocaleUtils.qml

```diff
@@ -25,20 +25,28 @@ QtObject {
         return num.toLocaleString(locale, 'f', precision)
     }
 
-    function currencyAmountToLocaleString(currencyAmount, locale) {
-        if (!locale) {
-            console.log("Unspecified locale for: " + JSON.stringify(currencyAmount))
-            locale = Qt.locale()
+    function numberFromLocaleString(num, locale = null) {
+        locale = locale || Qt.locale()
+
+        return Number.fromLocaleString(locale, num)
+    }
+
+    function currencyAmountToLocaleString(currencyAmount, options = null, locale = null) {
+        locale = locale || Qt.locale()
+
+        if (!currencyAmount) {
+            return "N/A"
         }
         if (typeof(currencyAmount) !== "object") {
             console.log("Wrong type for currencyAmount: " + JSON.stringify(currencyAmount))
+            console.trace()
```

Comment:
Is this leftover from debugging, or you consider to be useful? if yes then `console.warn` will be a better fit above

Reply:
So, the log itself will not tell you which piece of code called this function the wrong way, unless you manually include that info in the parameters which kind of defeats the purpose. The trace will help you quickly find that.

Reply:
Does `console.warn` include that? I'm just used to using `console.log`

Reply:
There's no difference -> no automatic trace

Reply:
> Does `console.warn` include that? I'm just used to using `console.log`

Doesn't include a trace but it logs a warning message, in the logging category. It can be used for filtering only warning logs and it is an appropriate labelling if the code path is not an usual path.

Reply:
Changed to `console.warn`